app/views/users/sessions.py

```diff
@@ -141,6 +146,69 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
```

Comment:
Since you pointed out, now I notice that your implementation is of session.title is actually wrong. A speaker may have more than one session. The way you have implemented, there will be repetitive entries for a speaker in the csv for every session. It shouldn't be like that. Test it like @niranjan94 suggested.